lib/suma/message/preferences.rb

```diff
@@ -57,6 +57,8 @@ def subscriptions
 
   def public_url = "#{Suma.app_url}/preferences-public?token=#{self.access_token}"
 
+  def preferred_language_name = Browser::AcceptLanguage.parse(self.preferred_language).first.name
```

Comment:
`Suma::I18n::SUPPORTED_LOCALES.fetch(self.preferred_language).language`

Better would be a method like `Suma::I18n.locale_or_default(self.preferred_language)` which always returned a locale if the preferred language is not set or is an invalid locale.